src/CplexSolverInterface.jl

```diff
@@ -8,6 +8,7 @@ type CplexMathProgModel <: AbstractMathProgModel
     branchcb
     incumbentcb
     infocb
+    solvetime
```

Comment:
This should probably be
```jl
solve time::Float64
```
(the constructor will have to be updated below, probably initialized to NaN)